rsa/key.py

```diff
@@ -570,81 +570,68 @@ def _save_pkcs1_pem(self):
         return rsa.pem.save_pem(der, b'RSA PRIVATE KEY')
 
 
-def find_p_q(nbits, getprime_func=rsa.prime.getprime, accurate=True):
+def find_p_q(nbits, getprime_func=rsa.prime.getprimebyrange, accurate="unused parameter retained for compatibility"):
     """Returns a tuple of two different primes of nbits bits each.
 
     The resulting p * q has exacty 2 * nbits bits, and the returned p and q
     will not be equal.
 
     :param nbits: the number of bits in each of p and q.
     :param getprime_func: the getprime function, defaults to
-        :py:func:`rsa.prime.getprime`.
+        :py:func:`rsa.prime.getprimebyrange`.
 
         *Introduced in Python-RSA 3.1*
 
-    :param accurate: whether to enable accurate mode or not.
```

Comment:
Please keep this parameter description but include a not that it is no longer in use (and as of which version of the library).